src/main.ts

```diff
@@ -313,6 +319,29 @@ class NixInstallerAction {
     }
   }
 
+  async flakehub_login(): Promise<string> {
+    const netrc_path = `${process.env["RUNNER_TEMP"]}/determinate-nix-installer-netrc`;
+
+    const jwt = await actions_core.getIDToken("api.flakehub.com");
+
+    await writeFile(
+      netrc_path,
+      [
+        `machine api.flakehub.com login flakehub password ${jwt}`,
+        `machine flakehub.com login flakehub password ${jwt}`,
+      ].join("\n"),
+    );
+
+    actions_core.info("Logging in to FlakeHub.");
+    if (this.extra_conf?.includes("netrc-file")) {
```

Comment:
This will falsely flag if the user has `netrc-file` anywhere in their config, even at the end of a line. Could we use a regex to ensure it's start of the line?

Reply:
Ohp, that's great idea! Yes.

Reply:
fixed!